docs/pages/dev-guide/developer-guide.md

```diff
@@ -1002,14 +1002,19 @@ plans:
         tasks: [sidecar]
 ```
 
-The command definition for the sidecar task includes an environment variables, `PLAN_PARAMETER1` and `PLAN_PARAMETER2`, that are not defined elsewhere in the service definition. You can supply these parameters when the plan is initiated.  The parameters will be propagated to the environment of every task launched by the plan.
+The command definition for the sidecar task includes environment variables, `PLAN_PARAMETER1` and `PLAN_PARAMETER2`, that are not defined elsewhere in the service definition. You can supply these parameters when the plan is initiated.  The parameters will be propagated to the environment of every task launched by the plan.
 
 To initiate the plan, execute an HTTP POST request against the `/v1/plans/sidecar-example/start` endpoint with the header `Content-Type: application/json` set and a JSON body consisting of environment variable name/value pairs. For example:
 
 ```bash
 $ curl -X POST -H "Authorization: token=$DCOS_TOKEN" -H "Content-Type: application/json" --data '{"PLAN_PARAMETER1": "sidecar", "PLAN_PARAMETER2": "plan"}' $DCOS_URL/service/$FRAMEWORK_NAME/v1/plans/sidecar-example/start
```

Comment:
@susanxhuynh The REST API is very inconsistently documented in this doc.  Sometimes endpoints are documented, sometimes they aren't.  Can we make this complete and consistent?